projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -56,56 +56,27 @@ export interface SortOption {
   templateUrl: './record-search.component.html'
 })
 export class RecordSearchComponent implements OnInit, OnChanges, OnDestroy {
-  /**
-   * Page anchor to scroll on the top
-   */
-  @ViewChild('topScrollAnchor') topScroll: ElementRef;
-
-  /**
-   * Current selected resource type
-   */
-  @Input()
-  currentType: string = null;
-
-  /**
-   * Search query
-   */
-  @Input()
-  q = '';
-
-  /**
-   * Define the current record's page
-   */
-  @Input()
-  page = 1;
-
-  /**
-   * Define the number of records per page
-   */
-  @Input()
-  size = 10;
 
-  /**
-   * Defined the sort order
-   */
-  @Input()
-  sort: string = null;
-
-  /**
-   * If admin mode is disabled, no action
-   * can be done on a record, as add, update or remove.
-   */
-  @Input()
-  adminMode: ActionStatus = {
-    can: true,
-    message: ''
-  };
+  /** Page anchor to scroll on the top */
+  @ViewChild('topScrollAnchor') topScroll: ElementRef;
```

Comment:
I did not review this file as it seems that it has only be reformatted. Let me know if it is not the case.

Reply:
perfectly correct. I tried to have a better structure to more quickly find a function (grouping getter/setter, public function, private, ...)